client/src/containers/LoginFormContainer.tsx

```diff
@@ -3,13 +3,16 @@ import { LoginForm, LoginProps} from '../components/LoginForm/LoginForm';
 import { login } from '../actions/user';
 
 function mapStateToProps(_state: any): LoginProps {
```

Comment:
Please remove the underscore. it denotes the variable is unused (and excludes it from Typescript's usage checking)